csrankings-d.csv

```diff
@@ -111,6 +111,7 @@ Dan Jurafsky,Stanford University,http://web.stanford.edu/~jurafsky,uZg9l58AAAAJ
 Dan Kifer,Pennsylvania State University,http://www.cse.psu.edu/~duk17,7PQN6r4AAAAJ
 Dan Knights,University of Minnesota,http://www.knightslab.org,zTkLS_gAAAAJ
 Dan Li 0001,Tsinghua University,https://nasp.cs.tsinghua.edu.cn/lidan.html,NOSCHOLARPAGE
+Dan Li 0016,Sun Yat-sen University,http://sse.sysu.edu.cn/teacher/209,YYig2NUAAAAJ

```

Comment:
Appears to be EE, not clear if meets inclusion criteria. No indication as to rank, etc. in English.